_raml/apps/legal/examples/put_legal_terms_request.json

```diff
@@ -1,3 +1,5 @@
 {
```

Comment:
I suggest putting a hint together with the description of the PUT calls.
See this example (ignore the blog post):
https://github.com/ePages-de/epages-docs/pull/896
You can decide if you want the description also on the top, or only with the request content.
You might want to ask the Unais if there is an easy way to color the warning sign, (i.e. inject HTML-Element Style into the text, around the icon)